tests/unit/date-helper.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { shortPacificDate } from '../../src/utils/date-helper'
+
+describe('Date Helper', () => {
+  it('returns short pacific date shortPacificDate()', () => {
+    expect(shortPacificDate('2021-07-01T00:00:00+00:00')).toBe('June 30, 2021')
+  })
+
+  it('returns RangeError when empty string is passed shortPacificDate()', () => {
+    expect(() => shortPacificDate('')).toThrowError('Invalid time value')
```

Comment:
Do you think it would be better if the method returned null instead? (Do its user expect it to throw on error?)

Reply:
I did try that but get this. 

![image](https://github.com/bcgov/bcrs-shared-components/assets/116035339/cbabde79-e5bd-4629-a945-c923a1637713)

I am now thinking it might be the way function is written is the casue

Reply:
Yes. I was thinking that the function should be updated to handle this gracefully instead of throwing an exception. Thoughts?

Reply:
Yes function needs to be updated.